# This is a commanded velocity and yaw in free space
# sent to quadrotor from user

float64 vx  # Velocity in x direction
float64 vy  # Velocity in x direction
float64 vz  # Velocity in x direction
float64 yaw  # Commanded yaw
